app-arch/torcx/files/vendor.json

```diff
@@ -0,0 +1,11 @@
+{
+    "kind": "profile-manifest-v0",
+    "value": {
+        "images": [
+            {
+                "name": "docker",
+                "reference": "17.05.0"
```

Comment:
Noting that image symlinks don't work.  I get `apply failed: image docker:com.coreos.cl not found`.  It does work if I just copy the file to have the same name.